index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <script type="module" src="scripts/index.js"></script>
+    <title>Haker_news_JS</title>
+  </head>
+  <body>
```

Comment:
I was wondering if we could improve html structure a little bit:

`<body>
  <main>
      <header class="top-panel">
        <img alt="Hacker News Logo" src="images/logo.gif"/>
        <h1 class="logo">Hacker News</h1>
      </header>
      <section class="hacker-news">
          <ol id="news-list" class="hacker-news"></ol>
      </section>
    <aside>
      <form class="form" name="user"></form>
      <div id="userInfo" class="user-info"></div>
    </aside>
  </main>
</body>`

Reply:
Structure optimization. Tag IMG simplification as much as possible